libs/designsystem/src/lib/components/page/page.component.scss

```
@@ -24,12 +24,20 @@ ion-toolbar {
   --padding-bottom: 0;
 
   box-sizing: border-box;
-  padding-inline: utils.size('xs');
-  transition: padding-inline utils.get-transition-easing('enter-exit')
-    utils.get-transition-duration('short');
+
+  $button-padding: utils.size('xs');
+
+  padding-left: $button-padding;
+  padding-right: $button-padding;
```

Consider using a CSS custom property for button padding instead of two identically named Sass variables. 

```suggestion
  --button-padding: #{utils.size('xs')};

  padding-left: var(--button-padding);
  padding-right: var(--button-padding);
```

It will simplify overriding the values below in `&.content-scrolled`